docs/api/sovity-edc-api-wrapper.yaml

```diff
@@ -1351,6 +1351,7 @@ components:
     ContractAgreementTerminationInfo:
       required:
       - detail
+      - initiator
```

Comment:
since we do not have a fully managed "process" of termination, I would recommend that we do not coin a new term, and simply go by "terminatedBy" or sth like that.

No "terminators", please 😄 

We'll get DMCA'd

Reply:
Replaced by `terminatedBy` then.